core/healthcheck.go

```diff
@@ -44,15 +41,9 @@ func (d *portHealthChecker) Check(ctx context.Context) (err error) {
 		return nil
 	}
 
-	// show health-check logs in a --debug vertex
-	vtx := rec.Vertex(
-		digest.Digest(identity.NewID()),
-		strings.Join(args, " "),
-		progrock.Internal(),
-	)
-	defer func() {
-		vtx.Done(err)
-	}()
+	// always show health checks
```

Comment:
no-blocker, but are the health check intervals at the beginning slow enough to not like show 3 every time before either the check passes or exponential backoff grows enough to make them not noisy?

Reply:
Initial interval is 100ms, up to 1 minute, giving up after 15 mins total so shouldn't be anything too crazy.

In ad-hoc testing it's been pretty quiet, so it seems like a decent balance:

![image](https://github.com/dagger/dagger/assets/1880/8348d850-a7a9-4a19-9dfe-14339aaefe30)